src/redux/uniswap.js

```diff
@@ -29,16 +30,16 @@ const UNISWAP_LOADED_ALL_TOKENS = 'uniswap/UNISWAP_LOADED_ALL_TOKENS';
 
 const UNISWAP_UPDATE_FAVORITES = 'uniswap/UNISWAP_UPDATE_FAVORITES';
 const UNISWAP_CLEAR_STATE = 'uniswap/UNISWAP_CLEAR_STATE';
-
 // -- Actions --------------------------------------------------------------- //
 export const uniswapLoadState = () => async (dispatch, getState) => {
   const { network } = getState().settings;
   dispatch({ type: UNISWAP_LOAD_REQUEST });
   try {
     const favorites = await getUniswapFavorites(network);
     remove(favorites, address => toLower(address) === toLower(SOCKS_ADDRESS));
+    const lists = await getUniswapLists(network);
```

Comment:
let's get rid of lists in the uniswap redux state as it exists separately in redux userLists